examples/demos/dnd.js

```diff
@@ -55,7 +55,7 @@ class Dnd extends React.Component {
 
     const nextEvents = events.map(existingEvent => {
       return existingEvent.id == event.id
-        ? { ...existingEvent, start, end }
+        ? { ...existingEvent, start, end, allDay }
```

Comment:
doesn't the `existingEvent` already have this?

Reply:
The old story's `moveEvent` handler did't honor changes affecting `allDay`. This change purports to make it possible to demo moving events in and out of the allDay gutter - supported by DnD but previously not supported by the story as it ignored the new value for allDay.

Reply:
I should have done this as a separate PR for sure! Newbie/learning.